cves/kernel/CVE-2020-36558.yml

```diff
@@ -34,7 +34,7 @@ announced_instructions: |
   This is not the same as published date in the NVD - that is below.
 
   Please enter your date in YYYY-MM-DD format.
-announced_date: '2022-07-21'
+announced_date: '2020-02-18'
```

Comment:
Curious where you found this date? The lore date is Mon, 10 Feb 2020, https://lore.kernel.org/all/20200210190721.200418-1-edumazet@google.com/.

Not saying your wrong.

Reply:
Changed